tagged-localbuild.xml

```diff
@@ -1,731 +1,742 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <manifest>
-  <remote fetch="https://github.com/NXPNFCProject/" name="NXP"/>
-  <remote fetch="https://android.googlesource.com/" name="aosp"
-          review="https://android-review.googlesource.com/"/>
-  <remote fetch="https://github.com/exfatprogs/" name="exfatprogs"/>
-  <remote fetch="https://github.com/sonyxperiadev/" name="sony"/>
+  <remote name="NXP" fetch="https://github.com/NXPNFCProject/"/>
+  <remote name="aosp" fetch=".." review="https://android-review.googlesource.com/"/>
```

Comment:
fetch= should be as in the previous manifest, otherwise it will look for AOSP in github.com/ :)

Reply:
So repo does not check that? nice.

Reply:
repo just executes everything you specify in manifest. This adds flexibility to be able to do things like `repo init -u ../parentdir-localtest`